pipeline-scripts/buildlib.groovy

```diff
@@ -945,7 +966,16 @@ The reconciled (downstream OCP) Dockerfile can be viewed here:
  - https://pkgs.devel.redhat.com/cgit/${distgit}/tree/Dockerfile?id=${val.sha}
 
 Please direct any questions to the Automated Release Tooling team (#aos-art on slack).
-        """);
+        """
+        } else {
+            error("Unable to determine notification reason; something is broken"
```

Comment:
Needs a closing paren
Also, probably ought to tell them what to do about it... which I think is something along the lines of "hang tight and see if ART fixes it, but complain if it continues"

Reply:
Nice catch!
I've also updated the message.